src/header/internal.tsx

```diff
@@ -60,7 +60,7 @@ export default function InternalHeader({
             isRefresh && styles[`title-variant-${variantOverride}-refresh`]
           )}
         >
-          <HeadingTag className={clsx(styles.heading, styles[`heading-variant-${variantOverride}`])}>
+          <HeadingTag className={clsx(styles.heading, styles[`heading-variant-${variantOverride}`])} tabIndex={-1}>
```

Comment:
I am curious if it is safe to allow all headers to accept focus. Some tools might rely on the presence of tab-index -- would make sense to check how screen-readers react to that.